libkbfs/conflict_resolver.go

```diff
@@ -3043,6 +3066,12 @@ func (cr *ConflictResolver) completeResolution(ctx context.Context,
 		return err
 	}
 
+	defer func() {
```

Comment:
Actually, why don't I just move `doBlockPuts` call here -- I don't see any real downside to that.  Then I only need one cleanup call.